crates/bevy_winit/src/lib.rs

```diff
@@ -696,6 +676,44 @@ pub fn winit_runner(mut app: App) {
                 runner_state.active = ActiveState::WillSuspend;
             }
             Event::Resumed => {
+                #[cfg(any(target_os = "android", target_os = "ios", target_os = "macos"))]
```

Comment:
Creating windows outside of `Event::Resumed` is indeed incorrect.
So this change is great!

I would recommend you to move window creation of all targets inside `Event::Resumed`, it would cut down on the `cfg` guards.
Let me know if there is a reason why this can't be done or isn't desirable!